javascript-1/script4.js

```diff
@@ -0,0 +1,6 @@
+console.log("EXERCITIUL 4");
+
+let nume="Lazaroiu";
+let prenume="Razvan Gabriel";
+
+console.log("Numele meu este:Lazaroiu Razvan Gabriel");
```

Comment:
Si aici scopul exercitiului ar fi fost sa folosesti variabilele in interiorul textului astfel incat daca schimbi valoarea variabilelor sa iti se modifice si textul